html-css/week1/nadiia/portfolio/index.html

```diff
@@ -0,0 +1,298 @@
+      <div class="circle b171 a43"></div>
+      <div class="circle b194 a44"></div>
+      <div class="circle b194 a45"></div>
+      <div class="circle b164 a46"></div>
+      <div id="catleft">
+        <div id="cathead">
+          <div class="catear leftear"></div>
+          <div class="catear rightear"></div>
+          <div class="catshead"></div>
+          <div class="catnose"></div>
+          <div class="cateye lefteye"></div>
+          <div class="cateye righteye"></div>
+          <div class="catcheek rightcheek"></div>
+          <div class="catcheek leftcheek"></div>
+        </div>
+        <div id="catbody">
+          <div class="catbreastclose"></div>
+          <div class="catbreast"></div>
+          <div class="cattummyclose"></div>
+          <div class="cattummy"></div>
+          <div class="catpreleg"></div>
+          <div class="catleg"></div>
+          <div class="catpaw leftpaw"></div>
+          <div class="catpaw rightpaw"></div>
+        </div>
+        <div id="cattail">
+          <div class="cattailmiddle"></div>
+          <div class="cattailstart"></div>
+          <div class="cattailclose"></div>
+          <div class="cattailend"></div>
+        </div>
+      </div>
+      <div class="catright">
+        <div class="cathead">
+          <div class="catear leftear"></div>
+          <div class="catear rightear"></div>
+          <div class="catshead"></div>
+          <div class="catnose"></div>
+          <div class="cateye lefteye"></div>
+          <div class="cateye righteye"></div>
+          <div class="catcheek rightcheek"></div>
+          <div class="catcheek leftcheek"></div>
+        </div>
+        <div class="catbody">
+          <div class="catbreast"></div>
+          <div class="cattummy"></div>
+          <div class="catbreastclose"></div>
+          <div class="cattummyclose"></div>
+          <div class="catpreleg"></div>
+          <div class="catleg"></div>
+          <div class="catpaw leftpaw"></div>
+          <div class="catpaw rightpaw"></div>
+        </div>
+        <div class="cattail">
+          <div class="cattailmiddle"></div>
+          <div class="cattailstart"></div>
+          <div class="cattailclose"></div>
+          <div class="cattailend"></div>
+        </div>
+      </div>
+      <div class="welcome">
+        <p class="maintext">Hello, I'm Nadiia</p>
+        <p class="undertext">a designer, artist and web-developer</p>
+      </div>
+    </header>
+    <main>
+      <p class="art_project" id="about">About myself</p>
+      <p class="undertext plus">
+        I'm a person, who thinks about many things and shares them, providing my
+        ideas to web-sites or drawings. I like to express my feelings, emotions
+        and thoughts through canvas and it doesn't matter, if this canvas is on
+        a computer or on the table. If I have a free time, I prefer spending it
+        improving my skills.
+      </p>
+      <p class="art_project">Technical skills</p>
```

Comment:
This would be an `<h3>` as it's part of the "About me" section and doesn't have it's own menu item. Basically, it's going into detail in "About me" where you've already used an `<h2>`.
```suggestion
      <h3 class="section-title">Technical skills</h3>
```

The class name of `art_project` might not be the most fitting here. I'd suggest `section-title` or `section-headline`.